bin/run_benchmark.py

```diff
@@ -156,10 +169,19 @@ def is_terminated(run_id: str) -> bool:
 
 def is_failed(run_id: str) -> bool:
     print(run_id, f"Checking benchmark run success/failure")
-    info = get_run_info(run_id)
+    info = get_run_stats(run_id)
     response = json.loads(info.read())
+    stats = response["statistics"]
     pprint.pprint(response)
-    return response["errors"] is not None and len(response["errors"]) > 0
```

Comment:
errors are common and expected, and they include jitter warnings, etc, so `failedSLAs` is the correct field.

Reply:
`failedSLAs` is in the response from?

Reply:
ah, from hyperfoil